### ISSUE TITLE
Graph chars size

### ISSUE THREAD

**Issue by Aditi00a** [JohnSnowLabs/spark-nlp#8274]
## Description
training a NER model using NerDLApproach but encountering this error 
pyspark.sql.utils.IllegalArgumentException: requirement failed: Graph chars size should be 219: Could not find a suitable tensorflow graph for embeddings dim: 768 tags: 23 nChars: 219.

## Expected Behavior
it should train the model

## Current Behavior
training a NER model using NerDLApproach but encountering this error 
pyspark.sql.utils.IllegalArgumentException: requirement failed: Graph chars size should be 219: Could not find a suitable tensorflow graph for embeddings dim: 768 tags: 23 nChars: 219.
with 1500 training data rows in iob format

does this issue occur because of large training data size or large chunks ? how do i resolve it ?

## Context
unable to train a NER model

## Your Environment
Spark NLP version sparknlp.version():
Spark NLP version: 3.3.1

Apache NLP version spark.version:
Apache Spark version: 3.0.3

Setup and installation (Pypi, Conda, Maven, etc.):
google colabs
!wget http://setup.johnsnowlabs.com/colab.sh -O - | bash

**Comment by maziyarpanahi** [comment]
Hi,

It's not about the size (number of rows) but the number of unique characters, labels, or embeddings dimensions. The error also indicates that you need to create a new TF graph based on those new numbers.

- This is a great tutorial on how to train the NerDL model: https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Public/4.NERDL_Training.ipynb
- This is a great tutorial on how to create a custom NerDL graph and use it to train on datasets where default graph files are not suitable: https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Public/4.1_NerDL_Graph.ipynb

**Comment by Aditi00a** [comment]
can this setGraphFolder() read directly from an s3 location on aws ?

**Comment by maziyarpanahi** [comment]
Please have a look at our documentation and examples, we have notebooks for everything: https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/jupyter/training/english/dl-ner/mfa_ner_graphs_s3.ipynb

**Comment by Aditi00a** [comment]
> Please have a look at our documentation and examples, we have notebooks for everything: https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/jupyter/training/english/dl-ner/mfa_ner_graphs_s3.ipynb

when i load a trained model via NerDLModel.load("model_file") do i need to pass the same graphfolder ? or it will work without it ?
